core/commonTest/src/kotlinx/serialization/features/ThirdPartyGenericsTest.kt

```diff
@@ -0,0 +1,70 @@
+package kotlinx.serialization.features
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.modules.contextual
+import kotlinx.serialization.serializer
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+open class ThirdPartyGenericsTest {
```

Comment:
Why this class needs to be `open`?

Reply:
Because `JvmThirdPartyGenericsTest` extends it.